src/inc/corinfo.h

```diff
@@ -493,7 +493,8 @@ enum CorInfoHelpFunc
 
     /* GC support */
 
-    CORINFO_HELP_STOP_FOR_GC,       // Call GC (force a GC)
+    CORINFO_HELP_STOP_FOR_GC,           // Call GC (force a GC)
+    CORINFO_HELP_STOP_FOR_GC_POP_FRAME, // Call GC (force a GC) and pop the current frame from the thread
```

Comment:
You should not need to ID for the new helper here. The JIT does not call it.

(Also, to add a new ID here - you would need to change the GUID on the JIT/EE interface and follow the rest of the ceremony with it.)

Reply:
Ok, thanks for letting me know